src/components/executive-education-2u/styles/_ExecutiveEducation2UErrorPage.scss

```diff
@@ -0,0 +1,50 @@
+.executive-education-2u-error-illustration {
+    align-items: center;
+}
+
+.executive-education-2u-error-heading {
+    line-height: 60px;
+
+    .executive-education-2u-error-heading-red {
+        font-size: 3.75rem;
+        font-weight: 700;
+        color: $brand-500;
+        margin-right: 10px;
+    }
+
+    .executive-education-2u-error-heading-black {
+        font-size: 3.75rem;
+        font-weight: 700;
+        color: $dark-700;
+    }
+}
+
+.executive-education-2u-error-heading-details {
+    align-items: center;
+    font-family: $font-family-monospace;
+    font-size: 1.125rem;
+    font-weight: 400;
+    line-height: 28px;
+    color: $gray-700;
+}
+
+.executive-education-2u-error-link-description {
+    align-items: center;
+    font-family: $font-family-monospace;
+    font-size: 0.875rem;
+    font-weight: 400;
+    line-height: 24px;
+    padding-right: 8px;
+    color: $gray-500;
+}
+
+.executive-education-2u-error-link-box {
+    border: 1px solid $gray-300;
+    border-radius: 6px;
+    height: 24px;
+    padding-top: 6px;
+    padding-bottom: 8px;
+    font-size: 0.75rem;
```

Comment:
Consider using the `.x-small` class name or the `$font-size-xs` SCSS instead of hardcoding the font-size.